lib/matplotlib/image.py

```diff
@@ -28,6 +28,8 @@
     Affine2D, BboxBase, Bbox, BboxTransform, BboxTransformTo,
     IdentityTransform, TransformedBbox)
 
+import matplotlib.image as mimage
```

Comment:
This is a very confusing import in this file... this file IS `matplotlib.image`

You should be able to remove the `mimage.` below, as `AxesImage` is defined in this file and therefore exists in the namespace already.

Reply:
Hi @ksunden, thank you for pointing out this confusing line. I have made small adjustments in the most recent update. Please let me know if it looks good. Thank you for your time!